client/post-editor/editor-gutenberg-opt-in-sidebar/style.scss

```diff
@@ -0,0 +1,5 @@
+.editor-gutenberg-opt-in-sidebar {
+	text-align: center;
+	font-weight: bold;
+	padding: 10%;
+}
```

Comment:
Don't forget EOL for newfiles. Looks like we're missing a few.

Reply:
Weird, linter doesn't catch these? I didn't know we had a guideline.

Reply:
It's not enforced, generally I'll only leave a minor comment when I see em in the diff :)